lib/elixir/lib/list.ex

```diff
@@ -768,8 +772,9 @@ defmodule List do
   Converts a charlist to an existing atom. Raises an `ArgumentError`
   if the atom does not exist.
 
-  Currently Elixir does not support conversions from charlists
-  which contains Unicode codepoints greater than 0xFF.
+  As of Erlang/OTP 20, Elixir supports conversions from charlists which contains
```

Comment:
Since we support Erlang/OTP 20+ only, I believe we can just say "Elixir supports conversions from charlists which contains any Unicode codepoint.".

Reply:
So no mention to earlier version?